sirepo/pkcli/flash.py

```diff
@@ -53,14 +54,12 @@ def _remote_file(dest):
     d = sirepo.pkcli.admin.proprietary_code_dir(t)
     pkio.mkdir_parent(d)
     s = sirepo.sim_data.get_class(t)
-    for e in simulation_db.examples(t):
-        _get_file(d.join(s.proprietary_lib_file_basename(e)))
+    _get_file(d.join(s.FLASH_RPM_FILENAME))
```

Comment:
Use `urllib.request.urlretrieve` instead of `_get_file`.

I think this would always be a url for convenience of the code. Therefore, you can't really use py.path on it. 

Let's talk about this over video.